--- Segmentfault_old.c	2014-05-07 16:57:45.416708227 +0800
+++ Segmentfault_new.c	2014-05-07 16:57:48.996708375 +0800
@@ -10,18 +10,19 @@
 int main(){
 	int i = 10;
 	//输出方式的错误
-	printf("%s\n", i);
+	printf("%d\n", i);
 	int test[1];
 	//往受到系统保护的内存地址写数据
-	scanf("%d",test[0]);
+	scanf("%d",&test[0]);
 	//数组越界
-	printf("%d\n", test[1]);
+	printf("%d\n", test[0]);
 	//地址错误
-	char *a="I am a teacher.";
-	char *b="You are a student.";
+	char a[]="I am a teacher.";
+	char b[]="You are a student.";
 	printf("初始化后的字符串：\n%s\n%s\n",a,b);
 	copyString(a,b);
 	printf("复制后的字符串：\n%s\n%s\n",a,b);
+	return 0;
 }
 
 
